libs/barista-components/secondary-nav/src/secondary-nav.scss

```diff
@@ -108,84 +149,72 @@ $dt-secondarynav-header-padding-left: 20px;
 }
 
 .dt-secondary-nav-link {
-  color: $gray-700;
-  font-size: 14px;
-  line-height: 30px;
-  display: block;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
   text-decoration: none;
+  outline: none;
+  height: 56px;
+  font-size: 15px;
+  padding: 8px 0 8px 20px;
+  border-width: 1px 0 0;
+  border-style: solid;
+  border-color: $gray-200;
+  font-family: inherit;
+  color: inherit;
```

Comment:
I guess there is no need to set this to inherit.

Reply:
fixed, rule was not needed